Tasks/ANT/anttask.ts

```diff
@@ -1,15 +1,18 @@
 /// <reference path="../../definitions/vsts-task-lib.d.ts" />
 
 import tl = require('vsts-task-lib/task');
+import path = require('path');
```

Comment:
make sure we have unified "File Path separator" or we should support both *nix/Windows file separators in nodejs